libs/api/domains/national-registry/src/lib/nationalRegistry.service.ts

```diff
@@ -1,166 +1,77 @@
-import { User } from '@island.is/auth-nest-tools'
 import type { Logger } from '@island.is/logging'
 import { LOGGER_PROVIDER } from '@island.is/logging'
-import { FeatureFlagService, Features } from '@island.is/nest/feature-flags'
-import { Inject, Injectable, NotImplementedException } from '@nestjs/common'
+import { Inject, Injectable } from '@nestjs/common'
 import { Birthplace, Citizenship, Housing, Spouse } from './shared/models'
 import { Name } from './shared/models/name.model'
 import { SharedPerson } from './shared/types'
-import { SoffiaService } from './v1/soffia.service'
 import { BrokerService } from './v3/broker.service'
 
-type ApiVersion = 'v1' | 'v3'
-
 @Injectable()
 export class NationalRegistryService {
   constructor(
-    private readonly v1: SoffiaService,
     private readonly v3: BrokerService,
-    private readonly featureFlagService: FeatureFlagService,
     @Inject(LOGGER_PROVIDER) private readonly logger: Logger,
   ) {}
 
-  async getApi(user: User, api: ApiVersion = 'v1') {
-    if (api === 'v3') {
-      return api
-    }
-
-    const disableSoffia = await this.featureFlagService.getValue(
-      Features.disableSoffia,
-      false,
-      user,
-    )
-    if (disableSoffia) {
-      return 'v3'
-    } else {
-      return api
-    }
-  }
-
-  getPerson(nationalId: string, api: ApiVersion = 'v1', useFakeData?: boolean) {
-    return api === 'v3'
-      ? this.v3.getPerson(nationalId, undefined, useFakeData)
-      : this.v1.getPerson(nationalId)
+  getPerson(nationalId: string, useFakeData?: boolean) {
+    return this.v3.getPerson(nationalId, undefined, useFakeData)
   }
 
   getChildCustody(nationalId: string, data?: SharedPerson) {
-    if (data?.api === 'v3') {
-      return this.v3.getChildrenCustodyInformation(
-        nationalId,
-        data?.rawData,
-        data?.useFakeData,
-      )
-    }
-
-    return this.v1.getChildCustody(nationalId, data?.rawData)
+    return this.v3.getChildrenCustodyInformation(
+      nationalId,
+      data?.rawData,
+      data?.useFakeData,
+    )
   }
 
-  async getChildDetails(
-    nationalId: string,
-    api: ApiVersion,
-    useFakeData?: boolean,
-  ) {
-    if (api === 'v3') {
-      return this.v3.getChildDetails(nationalId, useFakeData)
-    }
-    return this.v1.getPerson(nationalId)
+  async getChildDetails(nationalId: string, useFakeData?: boolean) {
+    return this.v3.getChildDetails(nationalId, useFakeData)
   }
 
-  getCustodians(
-    nationalId: string,
-    userNationalId: string,
-    data?: SharedPerson,
-  ) {
-    return data?.api === 'v3'
-      ? this.v3.getCustodians(nationalId, data?.rawData)
-      : this.v1.getCustodians(nationalId, userNationalId, data?.rawData)
+  getCustodians(nationalId: string, data?: SharedPerson) {
+    return this.v3.getCustodians(nationalId, data?.rawData)
   }
 
   getParents(nationalId: string, data?: SharedPerson, userNationalId?: string) {
```

Comment:
userNationalId is unused
```suggestion
  getParents(nationalId: string, data?: SharedPerson) {
```